torch/ao/quantization/fx/prepare.py

```diff
@@ -1349,6 +1353,8 @@ def prepare(
         qconfig_dict: Any,
         is_qat: bool,
         node_name_to_scope: Dict[str, Tuple[str, type]],
+        sample_args: Optional[Tuple[Any]] = None,
+        sample_kwargs: Optional[Dict[Any, Any]] = None,
```

Comment:
I actually don't understand what `sample_kwargs` here means. Is this an alternative way for the user to provide example inputs? Maybe I'm missing something but I don't see where this is consumed

Reply:
this is used to address the cases when a function doesn't have required positional arguments, and only keyword arguments:
e.g.
```
def forward(self, input_a=None, input_b=None, input_c=None):
    return ...
```

the corresponding sample_args/sample_kwargs for the above would be:
sample_args = None,
sample_kwargs = {"input_a": 3, "input_b": 4}

I guess this is rare, these cases can also be supported with the sample_args as well, are we OK not to support this case?